GOVERNANCE.md

```diff
@@ -217,6 +257,27 @@ push commits, etc.), so what's the minimal amount is subjective, and there will
 be cases where collaborators disagree on whether a nomination should move
 forward.
 
+Refrain from discussing or debating aspects of the nomination process
+itself directly within a nomination private discussion or public issue.
+Such discussions can derail and frustrate the nomination and cause unnecessary
```

Comment:
feel free to disregard if that's a valid form of written English I'm not familiar with
```suggestion
Such discussions can derail and frustrate the nominator and cause unnecessary
```

Reply:
The original is probably more the intent because it implies that everyone involved gets frustrated, not just the nominator. The “frustration” is redundant with “friction“ on the next line.

```suggestion
Such discussions can derail the nomination and cause unnecessary
```